lib/torrent/scheduler/dispatch/peer.go

```diff
@@ -92,10 +92,14 @@ func (p *peer) touchLastPieceSent() {
 // peerStats wraps stats collected for a given peer.
 type peerStats struct {
 	mu                    sync.Mutex
-	pieceRequestsSent     int // pieces we requested from the peer
-	pieceRequestsReceived int // pieces the peer requested from us
-	piecesSent            int // pieces we sent to the peer
-	piecesReceived        int // pieces we received from the peer
+	pieceRequestsSent       int // pieces we requested from the peer
+	pieceRequestsReceived   int // pieces the peer requested from us
+	piecesSent              int // pieces we sent to the peer
+
+	// pieces we received from the peer that we didn't already have
+	goodPiecesReceived int
+	// pieces we received from the peer that we already had
+	duplicatePiecesReceived int
```

Comment:
Nit: proper grammar please (i.e. capitalize first letter and add period at the end).

Reply:
Sure. For the pre-existing comments above as well? Style seems somewhat mixed for end-of-line comments.